machiavelli-cards/src/main/java/pl/jellytech/machiavelli/cards/utils/ControllerUtils.java

```diff
@@ -0,0 +1,63 @@
+package pl.jellytech.machiavelli.cards.utils;
+
+import com.codahale.metrics.MetricFilter;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import pl.jellytech.machiavelli.cards.dtos.BaseResponseWrapper;
+import pl.jellytech.machiavelli.cards.dtos.ExceptionPayload;
+
+import java.sql.Timestamp;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.util.function.Supplier;
+
+@Slf4j
+public class ControllerUtils {
+    final static String requestTimerName = MetricRegistryToolsTypes.RequestTimer.name();
+
+    public static <T> ResponseEntity<BaseResponseWrapper<T>> SuccessResponse(T successPayload) {
+        final BaseResponseWrapper<T> response = new BaseResponseWrapper<T>(successPayload, null);
+        return ResponseEntity.status(HttpStatus.OK).contentType(MediaType.APPLICATION_JSON).body(response);
+    }
+
+    public static ResponseEntity<BaseResponseWrapper> ErrorResponse(Exception exception, HttpStatus status) {
```

Comment:
avoid method names with capital letter - it's not Java syntax
(and in this class you have both upper and lower camel case)